<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PopMiejsceWyjazdu</class>
 <widget class="QDialog" name="PopMiejsceWyjazdu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>286</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnDodajWyjazd">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbDodajWyjazd">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>90</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblDodajWyjazd">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>90</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lnlDodajWyjazd">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDodajWyjazdExit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>160</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
